src/components/PostPageDetail/index.js

```diff
@@ -84,8 +84,6 @@ const PostPageDetailIdComponent = (props) => {
   const [commentContext, dispatchComment] = React.useContext(Context).comments;
   const {comments} = commentContext;
   const [loadingGetComment, setLoadingGetComment] = React.useState(false);
-  const {updateVoteLatestChildrenLevel3, updateVoteChildrenLevel1, calculatePaddingBtm} =
-    usePostDetail();
   const {updateFeedContext} = usePostContextHook(contextSource);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateVoteLatestChildrenLevel3`, `updateVoteChildrenLevel1`, and `calculatePaddingBtm` functions from `usePostDetail` have been removed. If these functions were being used elsewhere in the component, make sure their functionality is covered by other parts of the code.
```diff
- const {updateVoteLatestChildrenLevel3, updateVoteChildrenLevel1, calculatePaddingBtm} = usePostDetail();
+ const {updateFeedContext} = usePostContextHook(contextSource);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->